localstripe/resources.py

```diff
@@ -244,6 +246,76 @@ def do_expand(path, obj):
         return obj
 
 
+class Balance(object):
+    object = 'balance'
+
+    def __init__(self, **kwargs):
+        if kwargs:
+            raise UserError(400, 'Unexpected ' + ', '.join(kwargs.keys()))
+
+        currencies = ['eur']
+
+        self.object = self.object
+        self.livemode = False
+        self.type = 'balance'
+        self.connect_reserved = []
+        self.available = [
+            {
+                'amount': 2000,
+                'currency': currency,
+                'source_types': {
+                    'card': 2000
+                }
+            }
+            for currency in currencies
+        ]
+        self.pending = [
+            {
+                'amount': 0,
+                'currency': currency,
+                'source_types': {
+                    'card': 0
+                }
+            }
+            for currency in currencies
+        ]
+
+        store[self.object] = self
+
+        schedule_webhook(Event('balance.available', self))
+
+    @classmethod
+    def _api_get(self):
+        obj = store.get(self.object, None)
+        if obj is None:
+            return self()
+        return obj
+
+    def _export(self, expand=None):
+        # expand is passed to be coherent with the server api
+        # but it's not used
+
+        obj = {}
+
+        for key, value in vars(self).items():
+            if not key.startswith('_'):
+                if isinstance(value, StripeObject):
+                    obj[key] = value._export()
+                elif (isinstance(value, list) and len(value) and
+                        isinstance(value[0], StripeObject)):
+                    obj[key] = [item._export() for item in value]
+                elif isinstance(value, dict):
+                    obj[key] = value.copy()
+                else:
+                    obj[key] = value
+
+        return obj
+
+
+extra_apis.append((
+    ('GET', '/v1/balance', Balance._api_get)))
```

Comment:
Can it be kept inside `server.py`? 
If think it's what you were proposing @adrienverge but I'm not sure.

Reply:
I'm OK if it stays here, as long as it's consistent. If it helps reducing the changes needed in `server.py`, why not.

Style notes:
- there are extra parentheses,
- it fits on one line.